tests/src/test/java/alluxio/client/cli/fsadmin/command/MetricsCommandIntegrationTest.java

```diff
@@ -25,6 +26,7 @@
  */
 public final class MetricsCommandIntegrationTest extends AbstractFsAdminShellTest {
   @Test
+  @Ignore("ALLUXIO-3226")
```

Comment:
@calvinjia Ignoring this test allowed a regression to slip through: https://github.com/Alluxio/alluxio/pull/7524. Can you work with @LuQQiu to get the test working? I don't think we should wait until ALLUXIO-3226 for this